tests/ra_aid/test_main.py

```
@@ -0,0 +1,123 @@
+        assert config["cowboy_mode"] is True
+        assert config["research_only"] is True
+        assert config["provider"] == "anthropic"
+        assert config["model"] == "claude-3-5-sonnet-20241022"
+        assert config["expert_provider"] == "openai"
+        assert config["expert_model"] == "gpt-4"
+        assert config["limit_tokens"] is False
+
+
+def test_temperature_validation(mock_dependencies):
+    """Test that temperature argument is correctly passed to initialize_llm."""
+    from ra_aid.__main__ import main, initialize_llm
```

Ruff (0.8.2)

`ra_aid.__main__.initialize_llm` imported but unused

Remove unused import: `ra_aid.__main__.initialize_llm`

(F401)